package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@uniswap/universal-router-sdk",
-  "version": "1.5.0-beta.2",
+  "version": "1.5.0-beta.3",
```

Comment:
in most of our repos the convention is to put new versions in their own commit

Reply:
Yeah I'll be doing that with non-beta versions, but the beta version that I'm putting out literally arent fully functional and are just for the NFT team to start integrations

Reply:
So I'm not tagging on github for those - Though maybe I should still put it in its own commit

Reply:
yah you don't necessarily need to tag them, but it's nice to have them in their own commits IMO